### DISCUSSION TITLE
AlDente 1.22.2

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1008] [General]
**"Fettuccine"**

- [All] Fixed a bug that caused some layouts to appear incorrect when using AlDente with right-to-left languages.
- [All] Fixed a bug that caused the VoiceOver Accessibility Feature to not work correctly.
- [Pro] Fixed a bug that caused the app to crash when deleting a schedule task in some cases.
- [Pro] Fixed a bug where the menu bar icon text was cut off at 100%.
- [Pro] Fixed a bug where Power Flow did not work as expected on Intel MacBooks with Thunderbolt 3 from 2016 and 2017.
- [Pro] Sailing Mode now persists a restart of AlDente or system reboot again.
- [Pro] Sleep is now disabled during all phases of Calibration Mode.
- [Pro] Fixed a bug that caused the battery percentage to always appear in the menu bar, even if disabled.

With peace & love from Vienna,  
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/davidwernhart/AlDente-Charge-Limiter/releases/tag/1.22.2'>AlDente 1.22.2</a>.</em>

**Comment by Trevorz101** [comment]
Feature Idea: Notified us when battery is falling below a set threshold as known it's best to keep battery between 20-80%.

**Comment by iats36** [comment]
i want to deploy aldente with script on jamf . can anyone help
